go.mod

```diff
@@ -246,6 +246,8 @@ replace (
 	// Required by Cortex https://github.com/cortexproject/cortex/pull/3051.
 	github.com/bradfitz/gomemcache => github.com/themihai/gomemcache v0.0.0-20180902122335-24332e2d58ab
 
+	github.com/prometheus/prometheus => github.com/vinted/prometheus v1.8.2-0.20220817132046-4af7e615cfb4
```

Comment:
Temporary fork until https://github.com/prometheus/prometheus/pull/11163 is figured out.